### ISSUE TITLE
Dev test: Accessible John travoltage 1.3.0-dev.20

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/tasks#813]
@terracoda @emily-phet @jhung @jobara @ariel-phet @phet-steele Accessibility features in john-travoltage are almost ready to be deployed as part of the main simulation.  Accessibility in that sim is "feature complete" and we are ready for a dev test before moving to an RC. 

**Link to Sim:** http://www.colorado.edu/physics/phet/dev/html/john-travoltage/1.3.0-dev.20/john-travoltage_en.html

@phet-steele could you please dev test the simulation as usual, and also test keyboard navigation and screen reader access in this sim on the following platforms

- Normal dev test:
  - [x] Chrome (latest) - normal dev test
  - [x] iOS (latest) - normal dev test

- Test keyboard navigation (without a screen reader)
  - [x] Safari
  - [x] Edge
  - [x] Firefox

- Test Screen reader access:
  - [x] Windows Firefox (latest) + NVDA
  - [x] OSX Safari (latest) + VoiceOver
  - [x] IE (11) + JAWS

Keyboard navigation without a screen reader should be consistent on all tested platforms.  When a screen reader is on, keyboard navigation and auditory descriptions could be quite different depending on which device you are testing.

If you find an overwhelming number of issues due to screen reader quirks or inconsistencies, please let me know and we can think about a better way to do this kind of test.  To gauge this, it might be helpful to place a limit on testing time for each device.

Regardless of inconsistencies, all content should still be discoverable.  For assistance, please see
http://www.colorado.edu/physics/phet/dev/html/john-travoltage/1.3.0-dev.20/john-travoltage-a11y-view.html

With a screen reader, you should be able to read all the content on the right while navigating the sim on the left.

Before testing with screen readers, please familiarize yourself with how to use one. The following information could be useful:

http://webaim.org/articles/jaws/
http://webaim.org/articles/nvda/
http://webaim.org/articles/voiceover/

In particular, each of the links has a section on useful key commands.

Before testing a sim, try reading a simple HTML page like this one: http://csb.stanford.edu/class/public/pages/sykes_webdesign/05_simple.html

Please let me know if you have any questions about the above test!  Assigning to @ariel-phet to prioritize and @emily-phet so that she is aware of this task.

**Comment by ariel-phet** [comment]
@phet-steele given the desire to publish a keyboard accessible version of this sim, making current top testing priority

**Comment by emily-phet** [comment]
@phet-steele Can you give a brief update of where this is at?

**Comment by phet-steele** [comment]
> @phet-steele Can you give a brief update of where this is at?

@emily-phet Well, it was given priority on Thursday night, so it's had one days worth of testing so far. So, not much progress yet.

**Comment by jessegreenberg** [comment]
@phet-steele I should also mention, the only screen reader/browser combinations that we we care about for right now are the ones listed in this issue ticket since each screen reader is not guaranteed to work well with anything else.  Sorry, I should have clarified that before.

**Comment by emily-phet** [comment]
@phet-steele Thanks! Are you finding issues? I was looking around to see if any had been posted and I can't find any.

**Comment by phet-steele** [comment]
@jessegreenberg @emily-phet sorry this took so long, there was certainly a learning curve and an amount of patience needed. I think this is done. Keyboard nav w/o a screen reader was universally good. Most issues were isolated to the JAWS + IE11 test case. The next test pass will certainly be helpful, and thank you @terracoda @jobara for logging some VO issues and helping the QA process.

**Comment by jessegreenberg** [comment]
Excellent, thanks for all the testing @phet-steele and the QA team! :)
